crds/operators.coreos.com_catalogsources.yaml

```diff
@@ -1025,6 +1025,9 @@ spec:
                         will be configured as if `restricted` was specified. Otherwise, it will be configured as if `legacy` was
                         specified. Specifying a value other than `legacy` or `restricted` result in a validation error. When using older
                         catalog images, which can not run in `restricted` mode, the SecurityContextConfig should be set to `legacy`.
+                        Important, Catalog Sources built with OPM versions < v1.21.0 will not be able to run in `restricted` mode, and should
+                        be set to `legacy`. Catalog Sources built with OPM versions >= v1.21.0 will be able to run in `restricted` mode,
+                        OLM releases >= v1.32.0 can only accept the restricted mode and will not accept the legacy mode.
```

Comment:
I found an issue that would be v0.32 and not v1.32
But I want the agreements and the right version to change this one.